aerogear/mobile-developer-console
# [WIP] show/hide links for SDK configuration steps based on bound services

## Motivation
Add references to relevant tickets or a short description about what motivated you do it. (E.g JIRA: https://issues.jboss.org/browse/AEROGEAR-8032)

## What
There should be links to SDK initialization

## How
Services info is passed to the client and based on which services are bound, the links are shown/hiden

## Verification Steps
 
1. Go to client App
2. On `Services` tab bind two services
3. Go to `Configuration` tab, you should see Service specific configuration steps for services you bound

## Checklist:


## Progress

- [x] Show based on bound services
- [ ] Links to docs based on current released version in environment variable
- [ ] View is according to flats 

## Additional Notes

WIP, waiting for the flats.
